src/main/java/snw/jkook/entity/channel/Channel.java

```diff
@@ -175,6 +184,15 @@ public int getRawAllow() {
         public int getRawDeny() {
             return rawDeny;
         }
+
+        @Override
+        public String toString() {
+            return "UserPermissionOverwrite{" +
+                    "userName=" + user +
```

Comment:
```suggestion
                    "user=" + user +
```
userName应该调用#getNickName

Reply:
6，这个好像是在我修正提交的时候弄混了，然而夏夜的要求是直接打印User对象